verilog/parser/verilog.y

```diff
@@ -5479,6 +5479,18 @@ continuous_assign
     { $$ = MakeTaggedNode(N::kContinuousAssign, $1, $2, $3, $4, nullptr); }
   ;
 
+net_alias_assign
+  : lpvalue '=' net_alias_assign
+    { $$ = ExtendNode($1, $2, $3); }
+  | lpvalue
+    { $$ = move($1); }
```

Comment:
The semantic action for the first element of a list should make a singleton list, like:
`{ $$ = MakeTaggedNode(kAliasNetLvalueList, $1); }`
(might need a new node enum)

however, with a doubleton as the base case:
`{ $$ = MakeTaggedNode(kAliasNetLvalueList, $1, $2, $3); }`
which is the same as creating a singleton list followed by `ExtendNode`.

Reply:
That makes sense, added the new node enum and refactored the code